CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - [#646](https://github.com/cosmos/iavl/pull/646) Remove the `orphans` from the storage
 
 - [#777](https://github.com/cosmos/iavl/pull/777) Don't return errors from ImmutableTree.Hash, NewImmutableTree, NewImmutableTreeWIthOpts
+- [#815](https://github.com/cosmos/iavl/pull/815) `NewMutableTreeWithOpts` was removed in favour of accepting options via a variadic in `NewMutableTree`
```

Comment:
What's the difference in the changelog between `Breaking Changes` and `API Changes`?